src/Sylius/Component/Mailer/Sender/Adapter/AbstractAdapter.php

```diff
@@ -11,23 +11,28 @@
 
 namespace Sylius\Component\Mailer\Sender\Adapter;
 
-use Sylius\Component\Mailer\Provider\DefaultSettingsProviderInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
  * @author Paweł Jędrzejewski <pawel@sylius.org>
+ * @author Jérémy Leherpeur <jeremy@leherpeur.net>
  */
 abstract class AbstractAdapter
```

Comment:
This looks for me like an useless duplication as renderer abstract class contains same code...